frontend/test/pytest/test_shotvector.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2023 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Integration tests for the runtime assertion feature."""
+
+import jax.numpy as jnp
+import pennylane as qml
+import pytest
+
+from catalyst import qjit
+
+
+class TestShotVector:
```

Comment:
Could you add a test for a shotvector like ((20,5), 100, (101,2)) that mixes shots with and without copies?

Reply:
Sure, I’ll add a test for it. Thanks for the suggestion!